plugins/org.jkiss.dbeaver.erd.ui/src/org/jkiss/dbeaver/erd/ui/part/AttributePart.java

```diff
@@ -289,4 +291,22 @@ public ConnectionAnchor getTargetConnectionAnchor(ConnectionEditPart connection)
     public ConnectionAnchor getTargetConnectionAnchor(Request request) {
         return new ChopboxAnchor(getFigure());
     }
+
+    @Override
+    protected AccessibleEditPart getAccessibleEditPart() {
+        if (this.accPart == null) {
+            this.accPart = new AccessibleGraphicalEditPart() {
+                public void getName(AccessibleEvent e) {
+                    e.result =
+                        "Attribute: " + ERDUIUtils.getFullAttributeLabel(getDiagram(), getAttribute(), true, true);
+                }
+
+                public void getDescription(AccessibleEvent e) {
+                    e.result = null;
```

Comment:
What's the point of returning `null` here?

Reply:
I'd remove this method. If you want to show that result is empty for description, probable, it's better to assign it to an empty string (IMHO)